src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -77,12 +79,78 @@ static ss::sstring describe_as_string(const T& t) {
     return ssx::sformat("{}", t);
 }
 
+// Kafka protocol defines integral types by sizes. See
+// https://kafka.apache.org/protocol.html
+template<typename T>
+constexpr int num_bits = CHAR_BIT * sizeof(T);
+
+template<typename T>
+constexpr bool is_short = std::is_integral_v<T>&& num_bits<T> < 32;
```

Comment:
nit: Formatting? Theres no whitespace before `&&`

Reply:
you could also add `!is_bool<T>` to the expression

Reply:
> nit: Formatting? Theres no whitespace before `&&`

Turns out, that is something the linter does.